pepper-apis/src/main/java/org/broadinstitute/ddp/model/dsm/ParticipantStatusTrackingInfo.java

```diff
@@ -212,43 +225,47 @@ public static RecordStatus figureOutStatus(
             } else if (delivered == null && received != null) {
                 return RecordStatus.RECEIVED;
             } else {
-                LOG.error("Something completely unexpected happened with ", entityName);
+                LOG.error("Something completely unexpected happened with kit");
```

Comment:
Any more information we can give here? This message could use more actionable information if we have it.

Reply:
Actually, I don't think we'll ever execute this branch. The other `if` branch conditions cover all possible cases. I don't actually know why we have this `else` branch. My purpose here is not to refactor/change implementation here so I had kept it the way it is.